UCLA Radio/UCLA Radio/data/RadioAPI.swift

```diff
@@ -87,7 +87,8 @@ class RadioAPI {
                 delegate?.failedToFetchData(error)
         }
     }
-    
+
+    static let updatedGiveawaysNotificationName = NSNotification.Name(rawValue: "DidFetchUpdatedTicketGiveaways")
```

Comment:
I think we should be using Notification instead of NSNotification, it's a little Swiftier. Noticed it here in particular, but I think there are a few occurrences throughout our code.